src/main/webapp/WEB-INF/css/style.css

```diff
@@ -0,0 +1,4 @@
+/*.selector-for-some-widget {*/
```

Comment:
Preco je css zakomentovane? Nevieme to nejako ukazkovo vyuzit aspon na zmenu farby textu?

Reply:
Pridal som ukazku - farbu textu ako si chcel, zatial aspon inline.

Problem je totiz v tom ze neviem prist na to ako povedat thymeleafu, kde je umiestnenie css a js separatnych suborov. Ak by si na to prisiel mozes to odkomentovat, ale zatial funguje iba inline pre obe.